cluster/gce/config-common.sh

```diff
@@ -60,3 +60,8 @@ if [[ "${FEDERATION:-}" == true ]]; then
 else
     NODE_SCOPES="${NODE_SCOPES:-compute-rw,monitoring,logging-write,storage-ro}"
 fi
+
```

Comment:
I'm not sure what is the policy of `common.sh` vs. `config-{default, test}.sh`. This looks reasonably but there is almost nothing in this file.

@mikedanese to clairfy